Added updater badge

rmkraus opened the pull request.

rmkraus committed:
Added updater badge

balloob commented on src/more-infos/more-info-updater.js:
```diff
@@ -1,20 +0,0 @@
-import Polymer from '../polymer';
-import { serviceActions } from '../util/home-assistant-js-instance';
```
Every HTML file needs a js file

rmkraus replied:
Added that file back in. Thanks.

rmkraus committed:
Style fixes and compile error fix to updater badge

1) Added two missing semicolons.
2) Added missing more-info js file for updater badge.

rmkraus committed:
Added minimum javascript to updater more-info
